### PR TITLE
Pv2 524 logging level pv2 510 nservice bus update

### PR DESCRIPTION
**Pull request by samuel-dev-dba**

### CODE REVIEW

File: src/SFA.DAS.Payments.AcceptanceTests.Components.Core/SFA.DAS.Payments.AcceptanceTests.Components.Core.csproj [hunk 1]
```diff
@@ -90,6 +90,7 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```
Comment (kuzinator) [inline src/SFA.DAS.Payments.AcceptanceTests.Components.Core/SFA.DAS.Payments.AcceptanceTests.Components.Core.csproj]: This is probably automatically added by IDE but, do we need app.config in this project?

File: src/SFA.DAS.Payments.EarningEvents.EarningEventsService/PackageRoot/Config/Settings.xml [hunk 2]
```diff
@@ -16,6 +16,8 @@
     <Parameter Name="SubscriptionName" Value="" />
     <Parameter Name="JobStatusQueueName" Value="" />
     <Parameter Name="TopicName" Value="" />
+    <Parameter Name="LoggingMinimumLevel" Value="" />
```
Comment (kuzinator) [inline src/SFA.DAS.Payments.EarningEvents.EarningEventsService/PackageRoot/Config/Settings.xml]: Can I suggest that this setting be called "LogLevel" instead? This seems to be the most common practice.

File: src/SFA.DAS.Payments.Application/SFA.DAS.Payments.Application.csproj [hunk 3]
```diff
@@ -23,4 +23,10 @@
     <ProjectReference Include="..\SFA.DAS.Payments.Messages.Core\SFA.DAS.Payments.Messages.Core.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```
Comment (kuzinator) [inline src/SFA.DAS.Payments.Application/SFA.DAS.Payments.Application.csproj]: What is this reference for? It gives me a warning when built.

File: src/SFA.DAS.Payments.Application/Infrastructure/Ioc/Modules/MessagingModule.cs [hunk 4]
```diff
@@ -35,6 +35,8 @@ protected override void Load(ContainerBuilder builder)
                 endpointConfiguration.UseSerialization<NewtonsoftSerializer>();
                 endpointConfiguration.EnableInstallers();
 
+                if(config.ProcessMessageSequentially) endpointConfiguration.LimitMessageProcessingConcurrencyTo(1);
```
Comment (kuzinator) [inline src/SFA.DAS.Payments.Application/Infrastructure/Ioc/Modules/MessagingModule.cs]: This is potentially dangerous. When you set this setting locally to "true", you have no reasons to change it to false and unintentionally your local environment has no parallel processing and is very different from cloud.

Comment (samuel-dev-dba) [reply src/SFA.DAS.Payments.Application/Infrastructure/Ioc/Modules/MessagingModule.cs]: True but very useful for dev debugging

File: src/SFA.DAS.Payments.AcceptanceTests.Components.Core/SFA.DAS.Payments.AcceptanceTests.Components.Core.csproj (hunk 1 above)
Comment (samuel-dev-dba) [reply src/SFA.DAS.Payments.AcceptanceTests.Components.Core/SFA.DAS.Payments.AcceptanceTests.Components.Core.csproj]: Removed

File: src/SFA.DAS.Payments.EarningEvents.EarningEventsService/PackageRoot/Config/Settings.xml (hunk 2 above)
Comment (samuel-dev-dba) [reply src/SFA.DAS.Payments.EarningEvents.EarningEventsService/PackageRoot/Config/Settings.xml]: Changed

File: src/SFA.DAS.Payments.FundingSource.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml [hunk 5]
```diff
@@ -5,6 +5,8 @@
     <Parameter Name="StorageConnectionString" DefaultValue="" />
     <Parameter Name="ServiceBusConnectionString" DefaultValue="" />
     <Parameter Name="LoggingConnectionString" DefaultValue="" />
+    <Parameter Name="LogLevel" DefaultValue="" />
```
Comment (abroodie) [inline src/SFA.DAS.Payments.FundingSource.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml]: Default here should be "Info" (or is it "Information")

File: src/SFA.DAS.Payments.ProviderPayments.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml [hunk 6]
```diff
@@ -9,6 +9,8 @@
     <Parameter Name="ServiceBusConnectionString" DefaultValue="" />
     <Parameter Name="LoggingConnectionString" DefaultValue="" />
     <Parameter Name="PaymentsConnectionString" DefaultValue="" />
+    <Parameter Name="LogLevel" DefaultValue="" />
```
Comment (abroodie) [inline src/SFA.DAS.Payments.ProviderPayments.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml]: Default here should be "Info" (or is it "Information")

File: src/SFA.DAS.Payments.RequiredPayments.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml [hunk 7]
```diff
@@ -9,6 +9,8 @@
     <Parameter Name="ServiceBusConnectionString" DefaultValue="" />
     <Parameter Name="LoggingConnectionString" DefaultValue="" />
     <Parameter Name="PaymentsConnectionString" DefaultValue="" />
+    <Parameter Name="LogLevel" DefaultValue="" />
```
Comment (abroodie) [inline src/SFA.DAS.Payments.RequiredPayments.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml]: Default here should be "Info" (or is it "Information")

File: src/SFA.DAS.Payments.FundingSource.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml (hunk 5 above)
Comment (samuel-dev-dba) [reply src/SFA.DAS.Payments.FundingSource.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml]: done

File: src/SFA.DAS.Payments.RequiredPayments.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml (hunk 7 above)
Comment (samuel-dev-dba) [reply src/SFA.DAS.Payments.RequiredPayments.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml]: done

File: src/SFA.DAS.Payments.ProviderPayments.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml (hunk 6 above)
Comment (samuel-dev-dba) [reply src/SFA.DAS.Payments.ProviderPayments.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml]: done

**Comment by ESFA-Pawan** [comment]
Created new pull request "Release/release 524 510 #132"

**Pull request closed without merging**